.github/workflows/icu_exhaustive_tests.yml

```diff
@@ -0,0 +1,155 @@
+# Copyright (C) 2016 and later: Unicode, Inc. and others.
+# License & terms of use: http://www.unicode.org/copyright.html
+#
+# GitHub Actions configuration for Exhaustive Tests for ICU.
+# 
+# Note: The exhaustive test configuration is in a separate file
+# so that it can be run independently from the regular builds.
+#
+# To run these tests, comment on a PR with "run exhaustive tests" 
+# or manually trigger the workflow. If triggered from a PR comment,
+# it adds checks to the PR and updates them with the results.
+
+name: Exhaustive Tests for ICU
+
+on:
+  workflow_dispatch:
+  issue_comment:
+    types: [created]
+
+permissions:
+  checks: write
+  pull-requests: write
+
+jobs:
+
+  # Runs exhaustive tests for ICU4J on Linux
+  icu4j-linux:
+    if: github.event.issue.pull_request != null &&
+        (contains(github.event_name, 'workflow_dispatch') ||
+         contains(github.event.comment.body, 'run exhaustive tests'))
+    runs-on: ubuntu-latest
+    timeout-minutes: 180
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      - name: Fetch PR Details
+        id: pr-details
+        if: contains(github.event.comment.body, 'run exhaustive tests')
```

Comment:
Does this mean that we will be able to put this string into a PR description, and it will run this workflow?

Reply:
Looks like it: https://github.com/orgs/community/discussions/25389